src/pipeline/src/manager.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use common_time::Timestamp;
+use datatypes::timestamp::TimestampNanosecond;
+
+use crate::table::PipelineTable;
+use crate::{GreptimeTransformer, Pipeline};
+
+pub mod error;
+pub mod pipeline_operator;
+pub mod table;
+pub mod util;
+
+const PIPELINE_TABLE_NAME: &str = "pipelines";
```

Comment:
What about moving this constant into `src/pipeline/src/manager/table.rs`? With the columns.